commands.js

```diff
@@ -1,77 +1,51 @@
 'use strict';
 const fs = require('fs');
-const chalk = require('chalk');
 const path = require('path');
 const os = require('os');
 
 const pathFile = getSavePath();
 
-function getSavePath() {
-  const currentDir = os.homedir();
-  const pth = process.env['SHORTC_PATH'] || currentDir;
-  return path.join(pth, 'shortc.json');
-}
-
-function createIfNotExists(pth) {
+const tryReadFile = async path => {
   try {
-    if (fs.existsSync(pth)) {
-      return;
-    } else {
-      fs.writeFileSync(pth, '[]');
-    }
-  } catch (error) {
-    console.log(error);
+    const file = await fs.promises.readFile(path, { encoding: 'utf8' });
+    return file;
+  } catch (err) {
+    if (err.code !== 'ENOENT') throw err;
+    await fs.promises.writeFile(path, '');
```

Comment:
This is strange. Why do we write file immediately after error?

Reply:
Because if the error is `ENOENT`, that means the file doesn't exists. But, it can be that the file can't be created due to another error, like access rights. So, we check for it here, by creating empty file.